documentation/3_Interoperability.md

```diff
@@ -0,0 +1,375 @@
+keeping track which data came from which Data Space and what the legal
+responsibility of handling this data is.
+
+Agreements between legal organizations managing a Data Space can reduce
+the burden on the participants by agreeing on legal equivalency of
+policies in both Data Spaces.
+
+## Interdependency models in Data Spaces
+
+As shown above, the burden of ensuring interoperability and the
+adherence to all rules of individual Data Spaces is in the responsibility
+of a participant. However, the complexity of achieving interoperability
+across multiple Data Spaces also greatly depends on how those are
+related.
+
+The simplest model is probably a hierarchical direct-dependency of
+Data Spaces. In a larger Data Space a smaller sub-Data Space could be
+created with additional rules, utilizing the governance model of the
+overarching Data Space, but introducing additional policies for the sub
+Data Space. E.g. think of an industrial Data Space where one participant
+wants to share data only with their direct suppliers instead of the
+entire Data Space. This can be realized as a separate Data Space or within
+the larger industrial Data Space by having an additional DGA with
+additional membership policies and specifying additional semantic models
+and processes. In our example one participant could specify that for a
+specific set of data other participants need to prove that they are
+suppliers of this participant and understand a specific semantic models
+and processes provided by this participant. This can be regarded as a
+hierarchical relationship between two Data Spaces. In this case
+interoperability should be straightforward to achieve.
+
+Another model is Data Space peers. Two Data Spaces operating in different
+domains, but with a substantial overlap of participants, which also
+require data from both Data Spaces for many use cases. To reduce the
+burden on participants guaranteeing interoperability, the two Data Spaces
+might agree on same requirements for protocols, semantic models and also
+on organizational processes, including agreements on legal equivalency.
+
+Last but not least, there is the case of completely unrelated Data Spaces
+where one or just a few participants have the need to access data from
+multiple Data Spaces. In this case the burden of interoperability fully
+lies on the participants as they will need to be able to comply with
+processes in both Data Spaces and potentially will need to provide
+completely separate technical environments to access data in different
+Data Spaces.
+
+No matter how Data Spaces are related and cross-Data Space
+interoperability will be achieved, it is always going to be the
+responsibility of the participant to keep track of which data was
+acquired through which Data Space and what obligations came with it.
+Especially if use cases need to combine data from different Data Spaces
+sophisticated data management will be required.
```

Comment:
This section calls for a figure on the different models described.

Reply:
any ideas and suggestions for figures are highly welcome :)

Reply:
Something in this direction? 
![image](https://github.com/International-Data-Spaces-Association/IDSA-Rulebook/assets/23654606/4ae3b461-88f6-47ce-b591-44a8f3020f1b)

Reply:
Maybe not a good idea to create a picture like this. It feels a little like islands. We want to show this more as a continuum.